### DISCUSSION TITLE
[V4] Will arbitrary properties support CSS variables without `var(...)`?

### DISCUSSION THREAD

**Discussion by ziloen** [tailwindlabs/tailwindcss#15191] [Ideas]
https://play.tailwindcss.com/XTVpl8VQI0
Arbitrary values supports CSS variables without `var(...)`, what about arbitrary properties?

**Comment by philipp-spiess** [comment]
These still require `[]` like they did before. So in your case:

```txt
[box-shadow:var(--shadow)] group-hover:[box-shadow:var(--shadow-hover)]
```

**Comment by julie598doyle** [comment]
Hello,
Arbitrary properties in CSS frameworks like Tailwind CSS allow you to use any CSS property, even those not predefined by the framework. However, they still require the var() function to use CSS variables within these properties.

Best Regards,
Julie Doyle
